build.sbt

```diff
@@ -2,35 +2,56 @@ import Tests._
 import de.heikoseeberger.sbtheader.HeaderPattern
 import com.typesafe.sbt.SbtScalariform
 import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+import sbt.Keys._
+import sbtassembly.AssemblyPlugin.autoImport._
+
+lazy val root = (project in file("."))
+  .enablePlugins(AutomateHeaderPlugin, SbtOsgi)
+  .dependsOn(cassandraLauncher % Optional)
+  .aggregate(cassandraLauncher)
```

Comment:
so the core project is still defined as "top level", hm, I find that confusing, could you move that to a separate  project?

please also verify that everything works with the release plugin